recipe/conda_build_config.yaml

```diff
@@ -1,4 +1,19 @@
-MACOSX_SDK_VERSION:            # [osx and x86_64]
-  - "10.13"                    # [osx and x86_64]
-MACOSX_DEPLOYMENT_TARGET:      # [osx and x86_64]
-  - "10.13"                    # [osx and x86_64]
+MACOSX_SDK_VERSION:            # [osx and x86_64 and qt_version!=6]
+  - "10.13"                    # [osx and x86_64 and qt_version!=6]
+MACOSX_DEPLOYMENT_TARGET:      # [osx and x86_64 and qt_version!=6]
+  - "10.13"                    # [osx and x86_64 and qt_version==none]
+  - "10.13"                    # [osx and x86_64 and qt_version==5]
+
+# https://doc.qt.io/qt-6/macos.html
+MACOSX_DEPLOYMENT_TARGET:      # [osx and qt_version == 6]
+  - '11.0'                     # [osx and qt_version == 6]
+MACOSX_SDK_VERSION:            # [osx and qt_version == 6]
+  - '12.3'                     # [osx and qt_version == 6]
+
+# As of 2024/03 -- OSX basically never had qt builds. If somebody asks, we can build them
+# As of 2024/03 -- qt6 works better than qt5 on aarch64. So drop qt5.
+# As of 2024/03 -- No windows users have expressed concerns about the qt libraries. don't build None
```

Comment:
If there's no inherent issue in building it, why _shouldn't_ we have a headless version also on windows?

Reply:
just trying to not have 96 builds going on at the same time.

Cutting the build matrix however i can.

Reply:
Fine by me. We can add headless on windows as soon as someone asks; perhaps by then we've unified the builds.

Reply:
right, or have dropped qt5.